Gruntfile.js

```diff
@@ -78,6 +78,15 @@ module.exports = function(grunt) {
           summary: true
         }
       },
+      consoleLoggingOptions: {
+        src: 'test/fixtures/pivotal/src/**/*.js',
+        options: {
+          specs: 'test/fixtures/pivotal/spec/*Spec.js',
+          helpers: 'test/fixtures/pivotal/spec/*Helper.js',
+          display: 'short',
+          showLogging: true
```

Comment:
I'm not sure this test actually does anything. Although you may be able to visually confirm that the logging works as expected, there is no expectation  in `test/fixtures/pivotal/spec/*Spec.js` that automatically verifies the new behavior.